servermon/hwdoc/migrations/0002_auto_20160614_1333.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
This migration is the reason the tests fail. Things are still a bit sketchy with the way we are going to handle the way to django 1.7 migrations. I would suggest merging this to 0001_initial for now